views/forms/signup.erb

```diff
@@ -68,7 +68,7 @@
 				<div class="step step--about-you step--all">
 					<h2 class="heading-medium">Your details</h2>
 					<%== input_for(@form, :person_name, label: 'Name') %>
-					<%== input_for(@form, :person_email, label: 'Email address', hint: 'Must be from a government organisation') %>
+					<%== input_for(@form, :person_email, label: 'Email address', hint: 'Must be from a government or public body') %>
```

Comment:
Maybe "government organisation or public body", otherwise it reads a bit like you have to represent a whole Government.